lib/postprocess/image.js

```diff
@@ -93,6 +95,19 @@ module.exports = async files => {
           response.headers['content-type'] || response.headers['Content-Type']
         imageExtensions = contentType.split('/')[1]
         imageBuffer = Buffer.from(response.data)
+
+        // check if the file is valid
+        try {
+          sizeOf(imageBuffer)
+        } catch (e) {
+          if (e instanceof RangeError) {
+            const imageSrc = fs.readFileSync(path.join(__dirname, 'missing-image.png')).toString('base64')
+            imageExtensions = 'png'
+            imageBuffer = Buffer.from(imageSrc, 'base64')
+          } else {
+            throw e

```

Comment:
We might consider using a different approach here - always replace the image with the placeholder as soon as rendering fails while logging the error.

Reply:
Good idea! Does it show up in CloudWatch if I just use console.log?